src/api/app/views/webui2/webui/project/subprojects.html.haml

```diff
@@ -7,53 +7,22 @@
     .row
       .col-md-12
         - if @parentprojects.present?
-          %h3 Parent projects of #{@project}
-          %table.responsive.table.table-striped.table-bordered.subproject-table
-            %thead
-              %tr
-                %th Parent Project
-                %th Title
-            %tbody
-              - @parentprojects.each do |project|
-                %tr
-                  %td.project_name= link_to(word_break(project.name), project_show_path(project))
-                  %td.project_title= word_break(project.title)
+          = render partial: 'subprojects_table', locals: { table_title: 'Parent', project: @project, projects: @parentprojects }
     .row{ class: ('mt-3' if @parentprojects.present?) }
       .col-md-12
-        %h3.d-md-inline-block Subprojects of #{@project}
         - if @subprojects.present?
-          %table.responsive.table.table-striped.table-bordered.subproject-table
-            %thead
-              %tr
-                %th Subproject
-                %th Title
-            %tbody
-              - @subprojects.each do |project|
-                %tr
-                  %td.project_name= link_to(word_break(remove_parent_name(project.name, @project.name), 50), project_show_path(project))
-                  %td.project_title= word_break(project.title)
+          = render partial: 'subprojects_table', locals: { table_title: 'Subproject', project: @project, projects: @subprojects }
         - else
           %p
             %i This project has no subprojects
         - if User.current.can_modify?(@project)
           = link_to('#', data: { toggle: 'modal', target: '#add-new-subproject-modal' }, title: 'Add New Subproject', class: 'nav-link') do
             %i.fas.fa-plus-circle.text-primary
             Add New Subproject
-
     .row.mt-3
       .col-md-12
         - if @siblings.present?
-          %h3 Sibling projects of #{@project}
-          %table.responsive.table.table-striped.table-bordered.subproject-table
-            %thead
-              %tr
-                %th Sibling Project
-                %th Title
-            %tbody
-              - @siblings.each do |project|
-                %tr
-                  %td.project_name= link_to(word_break(remove_parent_name(project.name, @parent_name), 50), project_show_path(project))
-                  %td.project_title= word_break(project.title)
+          = render partial: 'subprojects_table', locals: { table_title: 'Sibling', project: @project, projects: @siblings }
```

Comment:
Nitpick: s/Sibling/Siblings/

Reply:
This needs to be singular, so it should stay how it is.

Reply:
Ops. After reading Dany's comment I think it will be better to pass 'Sibling Projects' and remove the ending string 'Project' from the title of the table.

Reply:
> Ops. After reading Dany's comment I think it will be better to pass 'Sibling Projects' and remove the ending string 'Project' from the title of the table.

And after having a look at [the link](https://github.com/openSUSE/open-build-service/pull/6512#discussion_r239796592) you shared the current approach is fine for me.